staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -84,7 +84,6 @@ func (m *CRConverterFactory) NewConverter(crd *apiextensions.CustomResourceDefin
 	// Determine whether we should expect to be asked to "convert" autoscaling/v1 Scale types
 	convertScale := false
 	if utilfeature.DefaultFeatureGate.Enabled(apiextensionsfeatures.CustomResourceSubresources) {
-		convertScale = crd.Spec.Subresources != nil && crd.Spec.Subresources.Scale != nil
```

Comment:
accidental removal?

Reply:
v1 crd doesn't have subresource in the top, instead we only need to check the per-version subresource now